module-2/backend/week_two.md

```diff
@@ -5,34 +5,35 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
-2. Assume you have the following model:
-
-```ruby
-class Team << ActiveRecord::Base
-end
-```
-
-What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
-
-3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
-
-4. Assume that you added a line to your `Team` class as follows:
-
-```ruby
-class Team << ActiveRecord::Base
-  belongs_to :owner
-end
-```
-
-Now how would you find the owner of the team with an id of 4?
-
+  * allows us to make SQL queries to a database by writing Ruby
+2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
```

Comment:
This is a stale version of the questions. Be sure to set the Turing repo up as an [upstream repository](https://help.github.com/articles/configuring-a-remote-for-a-fork/), and then [fetch the latest version](https://help.github.com/articles/syncing-a-fork/) before you start working on the questions for the week.